sql/audit/repos-audit.sql

```diff
@@ -0,0 +1,87 @@
+/* 
+ * This pulls a list of all repositories in GHES with details on
+ * commit count, PR count, Issue count, Disk usage, Repo admins, Org owners, LFS usage, etc.
+ * Make sure to remove the limit for returning all results.
+ */
+
+SELECT 
+	repo.id as "Repo Id",
+	repo.owner_login as "Org Name",
+	repo.name as "Repository",
+	IFNULL(repo.active,0) as "is active",
+	IFNULL(commits.commit_count,0) as "Commit Count",
+	IFNULL(pr.count,0) as "PR Count",
+	IFNULL(prr.count,0) as "PR Review Count",
+	IFNULL(issue.count,0) as "Issue Count",
+	IFNULL(pb.branch_count,0) as "P Branch Count",
+	IFNULL(pb.branch_names,'') as "P Branch Names",
```

Comment:
❓ Is this PR branches or branches overall in the repo?

Reply:
Thanks for this catch, yes I added the correct description `Protected Branch Count` and `Protected Branch Names`